extensions/cornerstone/src/OHIFCornerstoneViewport.js

```diff
@@ -249,25 +249,40 @@ class OHIFCornerstoneViewport extends Component {
   render() {
     let childrenWithProps = null;
 
+    if (!this.state.viewportData) {
+      return <></>;
```

Comment:
Just `return null`. No need for a fragment here? Returning `null` makes React ignore the component, returning a fragment _might_ (?) incur some unneeded overheads.

Reply:
I think you're right. We have a lot of styling that targets diva inside named elements. Wasn't sure if this would have an impact. (Do fragments function as empty divs, or as fake containers to appease vdom?) I'll test with null, and if it works well, then I will go that route.